docs/verify_image.md

```diff
@@ -0,0 +1,133 @@
+# Verifying Images
+
+The RICOH THETA X can take 11K and 5.5 images.
+
+| model | resolution | width and height | approx megapixels | approx file size |
+| ---- | ------- | ------- | --------- | -------- |
+| THETA X | 11K |  11008 × 5504 | 60 megapixels | 9MB - 12MB  |
+| THETA X | 5.5K | 5504 × 2752 | 15 megapixels | 3.5 - 4.5MB |
+
+## File Size
+
+You can use the file size to quickly see if the expected resolution
```

Comment:
Missing period. Not critical.

Reply:
Obviously a small edit. I'll just fix small edits directly, normally. I was add "an edit" so that we'd have an easy example.

Reply:
let's keep practicing.  I'll do another pull request soon.